### PR TITLE
A few doc texts

### PR DESCRIPTION
**Pull request by stevsmit**
Preview: https://deploy-preview-42653--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-bug-fixes

Just some doc text.

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -1179,7 +1179,11 @@ This update corrects the accepted label specification so that the plug-in import
...
+* Previously, pre-flight checks did not account for {rh-openstack-first} resource utilization. As a result, those checks failed with an incorrect error message when utilization, rather than quota, impeded installation. Pre-flight checks now process both {rh-openstack} quota and utilization. The checks fail with correct error messages if the quota is sufficient but resources are not. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2001317[*BZ#2001317*])
+
+* Before this update, the oVirt Driver could specify ReadOnlyMany (ROX) and ReadWriteMany (RWX) access modes when creating a PVC from a configuration config file. This caused an error as the driver does not support shared disks, and so cannot support these access modes. With this update, the access mode has been limited to single node access. The system prevents any attempt to specify ROX or RWX when creating PVC and logs an error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1882983[*BZ#1882983*])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1184]: Should this be: 
```suggestion
* Before this update, the oVirt Driver could specify ReadOnlyMany (ROX) and ReadWriteMany (RWX) access modes when creating a PVC from a configuration file. This caused an error as the driver does not support shared disks, and so cannot support these access modes. With this update, the access mode has been limited to single node access. The system prevents any attempt to specify ROX or RWX when creating PVC and logs an error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1882983[*BZ#1882983*])
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -1271,7 +1275,9 @@ Previously, pre-flight checks did not account for {rh-openstack-first} resource
...
+* When using the OVN-Kubernetes network provider in {product-title} versions prior to 4.8, the node routing table was used for routing decisions. In newer versions of {product-title}, the host routing table is bypassed. In this release, you can now specify whether you want to use or bypass the host kernel networking stack for traffic routing decisions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1996108[*BZ#1996108*])
+
+* Previously, when Kuryr was used in a restricted installation with proxy, the Cluster Networking Operator was not enforfcing usage of the proxy to allow communication with the {rh-openstack-first} API. Consequently, cluster installation did not progress. With this update, the Cluster Network Operator can communicate with the {rh-openstack} API through the proxy. As a result, installation now succeeds. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1985486[*BZ#1985486*])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1280]: s/enforfcing/enforcing

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -1179,7 +1179,11 @@ This update corrects the accepted label specification so that the plug-in import
...
+
+* Before this update, the oVirt Driver could specify ReadOnlyMany (ROX) and ReadWriteMany (RWX) access modes when creating a PVC from a configuration config file. This caused an error as the driver does not support shared disks, and so cannot support these access modes. With this update, the access mode has been limited to single node access. The system prevents any attempt to specify ROX or RWX when creating PVC and logs an error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1882983[*BZ#1882983*])
+
+* Previously, disk uploads in the Terraform provider were not handled properly. As a result, the OpenShift installer failed. With this update, disk upload handling has been fixed, and disk uploads succeed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917893[*BZ#1917893*])
```
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:1186]: I think this should be the OpenShift installation program per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#_i).

**Review by opayne1 — COMMENTED** [review]
Just a few nits but otherwise looks good!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 3 above)
Comment (kalexand-rh) [reply release_notes/ocp-4-10-release-notes.adoc:1186]: {product-title} installation program, but yes.

**Pull request merged**